students/12th/juheejang/login.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="wrapLogin">
+      <div class="wrapLoginMain">
+        <div class="logoImage">
+          <img src="img/logo_text.png" alt="instagram logo" width="175" height="51">
+        </div>
+        <form>
+          <input type="text" id="userAcount" name="userAcount" placeholder="전화번호, 사용자 이름 또는 이메일" style="width: 258px; height: 36px;">
+          <br>
+          <input type="password" id="userPassword" name="userPassword" placeholder="비밀번호" style="width: 258px; height: 36px;">
```

Comment:
```suggestion
          <input type="text" class="userAcount" name="userAcount" placeholder="전화번호, 사용자 이름 또는 이메일">
          <input type="password" class="userPassword" name="userPassword" placeholder="비밀번호">
```
- 가급적 id 셀렉터 대신 class 셀렉터를 사용해주세요!
- 스타일은 inline 말고 css로 해주는게 좋습니다 👍 
- 간격을 띄우는 건 br 태그 대신 margin 등의 속성을 활용하면 좋겠죠?

Reply:
확인하였습니다 😀